### PR TITLE
Show section menu on each page

### PR DESCRIPTION
**Pull request by hikaruit15**
#348 
Fix format heading on pages to show section menu.

### CODE REVIEW

File: pages/firststeps.md [hunk 1]
```diff
@@ -8,37 +8,37 @@ We also want you to keep us updated a relatively good amount in the [Gitter chat
...
+##Step #0: Introductory Steps:
 1. Please send us your resume again (along with a short description of yourself and what you would like to be called) to vi@ole.org.
 2. Watch [Open Learning Exchange](https://github.com/open-learning-exchange/open-learning-exchange.github.io) and Follow [Emily](https://github.com/EmilyLarkin), [Stuart](https://github.com/sente) and [Dogi](https://github.com/dogi) on GitHub. In GitHub we "Watch" organization repositories and "Follow" individuals. When you are on a repository or profile page, look for these buttons in the top right of the screen. Please also follow each other to see what others in the group are doing.
 3. We use Crowdin to translate our user interface in many different languages. If you know another language, please go to [this Crowdin link](https://crowdin.com/project/open-learning-exchange/invite) to join Crowdin by creating an account. It would be very helpful for you to translate some of the words and familiarize yourself with the software. It would be great to get at least 10% on a language that you speak. If you want to spend additional time, feel free to provide translations to 100%.
 
 ###Step #1: BeLL Installation:
```
Comment (aberdean) [inline pages/firststeps.md]: Step 1 should also be an h2 heading.

File: pages/firststeps.md [hunk 2]
```diff
@@ -8,37 +8,37 @@ We also want you to keep us updated a relatively good amount in the [Gitter chat
...
 
-###Step #3: Markdown and Fork Tutorial:
+##Step #3: Markdown and Fork Tutorial:
 Follow the instructions on the [GitHub and Markdown page](githubandmarkdown.md). Make sure that you've linked to your github.io and your personal .md page on the [Gitter chat](https://gitter.im/open-learning-exchange/chat) (&lt;username&gt;.github.io and &lt;username&gt;.github.io/#!pages/&lt;username&gt;.md).
 
-###Step #4: BeLL-Apps Tutorial:
+##Step #4: BeLL-Apps Tutorial:
 Follow the instructions under the [BeLL-Apps tab](bellapps.md).
 
 ###Step #5: GitHub Issues Tutorial:
```
Comment (aberdean) [inline pages/firststeps.md]: Step 5 should be an h2 heading, as well.

Comment (hikaruit15) [inline pages/firststeps.md]: Sorry about that, my mistakes. I will change them.

Comment (hikaruit15) [inline pages/firststeps.md]: @aberdean  I fixed it.

**Pull request merged**